hx_lti_assignment/templates/hx_lti_assignment/create_new_assignment.html

```diff
@@ -197,7 +196,10 @@
 						<td class="field-target_object" >
 							<select id="id_assignmenttargets_set-{{forloop.counter0}}-target_object" name="assignmenttargets_set-{{forloop.counter0}}-target_object" class="selectpicker form-control" data-live-search="true">
 								{% for id,choice in form.fields.assignment_objects.choices %}
-								<option value="{{ id }}" {% if id == item.target_object.value %} selected {% endif %}>{{ choice }}</option>
+								{% target_object_is_in_course user id as exists %}
```

Comment:
These changes should probably be cut from the commit. We were trying to filter out the target objects that aren't in the active course. This has proved to be difficult, as target objects are not associated with any course. Likely this effort will be abandoned, and the 'promiscuous' functionality will be adopted. The reason we were doing this in the first place was because the tool was failing to add other courses' material to its own assignments, and we thought the bug was that the objects showed up at all, but in reality the bug is that they don't add cleanly.